File: pdp/KISIM2/src/RastgeleKisi.c
#include "../include/RastgeleKisi.h"

rskisi rskisiOlustur(){
   rskisi rs_kisi = (rskisi)malloc(sizeof(struct RASTGELEKISI));
   rs_kisi->im = imeiOlustur();
   rs_kisi->km = kimlikNoOlustur();
   rs_kisi->tf = telefonOlustur();
   rs_kisi->ys = YasOlustur(); 
   rs_kisi->kisi_ = kisiOlustur();
   return rs_kisi;

}
char* rskisi_uret(rskisi rsk, int kisiSayisi){

   FILE *fp = fopen("Kisiler.txt","w");

   for(int i = 0; i<kisiSayisi; i++){
      char* imei = imeino_uret(rsk->im);
      char* kimlik = kimlikno_uret(rsk->km);
      char* isim = kisi_uret(rsk->kisi_);
      int yas = Yasuret(rsk->ys);
      char* telefon = telefonNo_uret(rsk->tf);
	   fprintf(fp,"%s %s %d %s (%s)\n",kimlik,isim,yas,telefon,imei);
   }

	fclose(fp);
   return "";
}
void rskisiYoket(rskisi r){
   if(r == NULL) return;
	free(r);
	r = NULL;
}

File: pdp/KISIM1/RASTGELEKISIURET/src/rastgelekisiuret/RastgeleKisi.java
package rastgelekisiuret;

public class RastgeleKisi {
    Kisi YeniKisi = new Kisi();
    Yas YeniYas = new Yas();
    Telefon YeniTelefon = new Telefon();
    ImeiNo YeniImei = new ImeiNo();
    KimlikNo YeniKimlik = new KimlikNo();

    public String KisiOlustur() {
        String Gonder = YeniKimlik.KimlikNo() + " " + YeniKisi.KisiOlustur() + " " + YeniYas.YasOlustur()+ " " + YeniTelefon.TelefonNoUret() + " (" + YeniImei.ImeiNoUret() + ")";
        return Gonder;
    }

    public String KimlikKontrol(String Kimlik) {

        KimlikNo Tckontrol = new KimlikNo();
        return Tckontrol.KimlikNoKontrol(Kimlik);
    }

    public String ImeiKontrol(String Imei) {
        ImeiNo ImeiKontrol = new ImeiNo();
        return ImeiKontrol.ImeıKontrol(Imei);
    }
    
}
